packages/backend/src/controllers/githubController.ts

```diff
@@ -0,0 +1,182 @@
+import { ApiSuccessEmpty, ForbiddenError, GitRepo } from '@lightdash/common';
+import { createAppAuth } from '@octokit/auth-app';
+import { createTokenAuth } from '@octokit/auth-token';
+import { Octokit, Octokit as OctokitRest } from '@octokit/rest';
+import {
+    Controller,
+    Delete,
+    Get,
+    Middlewares,
+    OperationId,
+    Query,
+    Request,
+    Route,
+    SuccessResponse,
+} from '@tsoa/runtime';
+import express from 'express';
+import { getGithubApp, getOctokitRestForApp } from '../clients/github/Github';
+import { lightdashConfig } from '../config/lightdashConfig';
+import { githubAppService } from '../services/services';
+import { isAuthenticated, unauthorisedInDemo } from './authentication';
+
+const githubAppName = 'lightdash-dev';
```

Comment:
This is hardcoded here, not sure if we want to have it as ENV var already or will take care of it later since it isn't going to be used by anyone yet